src/main/java/onehundredandfour/GameHelper.java

```diff
@@ -0,0 +1,191 @@
+		Cards card = (Cards) deck.remove(0);
+		return card;
+		}
+	
+	public static Cards playerPlaysCard(List<?> hand, int location) {
+		Cards card = (Cards) hand.remove(location);
+		return card;
+		}
+	
+	public static List<Card> getPlayersCardsOnTable(List<Player> players) {
+		List<Card> card = new ArrayList<>();
+		for (Player player : players) {
+			Card c = player.getCardOnTable();
+			if (c != null) {
+				card.add(c);
+			}
+		}
+		return card;
+		}
+	
+	public static Card whoHasLowestCard(List<Card> p1) {
+		ArrayList<Card> cards = new ArrayList<>();
+		for (Card c : p1) {
+			cards.add(c);
+		}
+		Card min = cards.get(0);
+        for (Card i : cards){
+            min = min.getValue() < i.getValue() ? min : i;
+        }
+		return min;
+		
+	}
+	
+	public static Player getPlayerWithLowestCardOnTable(List<Player> players) {
+		List<Player> p = new ArrayList<>();
+		for (Player player : players) {
+			Card c = player.getCardOnTable();
+			if (c != null) {
+				p.add(player);
+			}
+		}
+		Player min = p.get(0);
+        for (Player i : p){
+            min = min.getCardOnTable().getValue() < i.getCardOnTable().getValue() ? min : i;
+        }
+		return min;
+		}
+	
+//	public static boolean checkCardAgainstRows(Card card, Game game) {
+//		List<Card> after = new ArrayList<>();
+//
+//		if (card.getValue() > game.getRow0().get(game.getRow0().size()-1).getValue()) {
+//		after.add(0, game.getRow0().get(game.getRow0().size()-1));	
+//		game.getRow0().add(card);
+//		}
+//		if (card.getValue() > game.getRow1().get(game.getRow1().size()-1).getValue()) {
+//			if (after.isEmpty()) {
+//				after.add(game.getRow1().get(game.getRow1().size()-1));	
+//			}else if (game.getRow1().get(game.getRow1().size()-1).getValue() > after.get(0).getValue()) {
+//				game.getRow0().remove(card);
+//				game.getRow1().add(card);
+//			}
+//		}
+//		if (card.getValue() > game.getRow2().get(game.getRow2().size()-1).getValue()) {
+//			if (after.isEmpty()) {
+//				after.add(game.getRow2().get(game.getRow2().size()-1));
+//			}else if(game.getRow2().get(game.getRow2().size()-1).getValue() > after.get(0).getValue()) {
+//				game.getRow0().remove(card);
+//				game.getRow1().remove(card);
+//				game.getRow2().add(card);
+//			}
+//		}
+//		if (card.getValue() > game.getRow3().get(game.getRow3().size()-1).getValue()) {
+//			if (after.isEmpty()){
+//				after.add(game.getRow3().get(game.getRow3().size()-1));	
+//			}else if (game.getRow3().get(game.getRow3().size()-1).getValue() > after.get(0).getValue()) {
+//				game.getRow0().remove(card);
+//				game.getRow1().remove(card);
+//				game.getRow2().remove(card);
+//				game.getRow3().add(card);
+//			}
+//		}
+//		if (!after.isEmpty()) {
+//			return true;
+//		}else
+//		{	System.out.println("Card is lower than last cards in all rows. \r\nPlease pick a row.");
+//			return false;
+//		}
+//	}
+	
+	public static boolean checkPlayerCardAgainstRows(Player pCard, Game game) {
```

Comment:
maybe some Javadoc would make it easier to understand what functionally happens here?